toolkits/springernature/packages/springernature-header/scss/50-components/core.scss

```diff
@@ -33,19 +33,19 @@
 }
 
 .c-header__journal-title {
-	@include media-query('tablet-vertical') {
+	@include media-query('tablet-wide-width') {
 		font-size: 2.2rem;
 	}
 
-	@include media-query('tablet') {
+	@include media-query('desktop-width') {
 		font-size: 2.7rem;
 	}
 
 	font-family: $context--font-family-serif;
 	line-height: 4.6rem;
 	font-size: 2rem;
 	font-size: 1.5rem;
```

Comment:
You have defined `font-size` twice here. Also, this is the `core` endpoint which means it needs to support _all_ browsers, we should not be using REMs for font-size. These should be pixel values that get updated to REM's in the `enhanced` endpoint

Reply:
It was there already but I can fix it sure. At least try

Reply:
Ok TBH we did not authored this package nor its latest changes. We are just patching colour and breakpoint keys in this PR. Hence, so far we do not use this package on the RP project (this will come).
Since I do not want to make false assumptions of what the pixel values... should be, I would rather leave @sky-jack taking care of this in a separate PR. What do you think @alexkilgour ?

Reply:
OK, that makes sense. Fine to leave that here. Leave it with me and I will talk to jack on Thursday